src/MIDebugEngine/AD7.Impl/AD7PendingBreakpoint.cs

```diff
@@ -34,7 +34,21 @@ internal class AD7PendingBreakpoint : IDebugPendingBreakpoint2
         private bool _deleted;
         private bool _pendingDelete;
 
-        public DebuggedProcess DebuggedProcess {  get { return _engine.DebuggedProcess;  } }
+        private IDebugFunctionPosition2 _functionPosition;
```

Comment:
Can we avoid storing the AD7 objects (`_functionPosition`, `_codePosition`, `_docPosition`) as member variables if we don't use them in GetDocumentContext?